lib/spack/docs/containers.rst

```diff
@@ -289,10 +306,6 @@ following ``Dockerfile``:
    COPY --from=builder /opt/view /opt/view
    COPY --from=builder /etc/profile.d/z10_spack_environment.sh /etc/profile.d/z10_spack_environment.sh
 
-   RUN yum update -y && yum install -y epel-release && yum update -y                                   \
```

Comment:
I won't miss this feature, but I see that @samcmill likes it.  If it goes back in it would be nice to be able to choose whether to update or not (I know, it's less simple but it's more reproducible, so choose your poison...).

Reply:
> If it goes back in it would be nice to be able to choose whether to update

Sure. That wouldn't be a big change. In case we also need a way to let the user specify which system package manager we are using - in case of a custom image that would be the simplest and more effective way to generate the lines. So something like:
```yaml
os_packages:
  command: yum
  update: False
  build:
  - libibverbs-dev
  final:
  - libibverbs
```
If we are using the images distributed on Docker Hub then the system package manager will be deduced from that like it is now.

Reply:
Sounds great!

Reply:
Done.